styles.css

```diff
@@ -181,12 +189,37 @@ article > div {
 	justify-content: space-around;
 }
 
+h3 {
+	margin-left: 10px;
+}
+
 
 /*footer*/
 footer {
 	background-color: #6B9080;
 	padding: 10px;
-	font-size: 10px;
+}
+
+.footer {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 50px;
+}
+
+footer h2 {
+	text-align: center;
+	padding-top: 80px;
+}
+
+.contact img{
+	height: 20px;
+	width: 100px;
+	padding-left: 10px;
+}
+
+.copyright {
```

Comment:
Under around 400px the copyright info begins starts to overflow its box, perhaps consider realigning it to the left at lower resolutions?